build.gradle

```diff
@@ -26,6 +26,7 @@ android {
 }
 
 dependencies {
+    api 'androidx.annotation:annotation:1.1.0'
```

Comment:
Note: I'm not sure if this is necessary in the normal way you folks build this project, but all the nonnull/nullable annotations in the source were problematic for me until I added this dependency.

Reply:
Should not be needed. This project is not using androidx yet.

Reply:
Hey guys, the androidx usage was inadvertently pushed into the master branch and has been reverted. If you rebase onto the current master, you will be able to remove the androidx dependency. Sorry for the confusion

Reply:
Fixed

Reply:
Awesome, thanks you. I will merge this right away, and my dangling androidx commit. As always, we appreciate the PR, and this should be released next Wednesday.